mapeditor/windownewmap.cpp

```diff
@@ -56,6 +58,19 @@ WindowNewMap::WindowNewMap(QWidget *parent) :
 		ui->cpuTeamsCombo->setItemData(i, QVariant(cpuPlayers.at(i)));
 	}
 	
+	// map option radios
+	connect(ui->mapOpt1, &QRadioButton::toggled, this, &WindowNewMap::on_mapOpt1_toggled);
+	connect(ui->mapOpt2, &QRadioButton::toggled, this, &WindowNewMap::on_mapOpt2_toggled);
+
+	on_mapOpt1_toggled(true);
+	on_checkSeed_toggled(false);
+
+	// size combo
+	connect(ui->sizeCombo, QOverload<int>::of(&QComboBox::activated), this, &WindowNewMap::on_sizeCombo_activated);
+	
+	// spin boxes
+	connect(ui->widthTxt, QOverload<int>::of(&QSpinBox::valueChanged), this, &WindowNewMap::on_widthTxt_valueChanged);
+	connect(ui->heightTxt, QOverload<int>::of(&QSpinBox::valueChanged), this, &WindowNewMap::on_heightTxt_valueChanged);
```

Comment:
I think Qt will automatically connect such signals with slots (since you're using `on_XXX_YYYChanged` form)

Reply:
Yes, seems it connected it automatically. I have removed all useless connects.